I_dont_have_vinaigrette.py

```diff
@@ -0,0 +1,43 @@
+import random
+import calendar
+import datetime
+
+
+
+def  print_random_date(s_date, e_date):
+    """
+    A function that prints a random date in range and if the date Monday prints I do not have a vinaigrette
+    :param s_date: s_date:start date (format YYYY-MM-DD)
```

Comment:
e_date and s_date are bad names-  dont use abbreviations - start date and end date -